step.yml

```diff
@@ -360,13 +360,15 @@ outputs:
       title: The created .xcarchive.zip file's path
       description: |-
          The created .xcarchive.zip file's path.
-  - BITRISE_XCODE_ARCHIVE_RAW_RESULT_TEXT_PATH:
-    opts:
-      title: The full, raw archive output file path
-      description: |-
-        This is the path of the raw archive results log file.
 
-        If `output_tool=xcpretty` and the archive fails this log will contain the archive output.
+  - BITRISE_XCODEBUILD_ARCHIVE_LOG_PATH:
+    opts:
+      title: "`xcodebuild archive` command log file path"
```

Comment:
`summary` or `description` would be nice for all inputs and outputs

Reply:
Agree, unfortunately only the Title is presented in the WFEditor, and in the case of Step Inputs, only the Title property is required: https://github.com/bitrise-io/stepman/blob/203f7a48d37aebc3cc5211690e9365f4d6d98d72/models/models_methods.go#L121-L162.

I will make up the missing description for the new outputs.

Reply:
Description added in https://github.com/bitrise-steplib/steps-xcode-archive/pull/238/commits/1d12da3b171083fc813e01d1f51aca6f63c8cc4d